airbyte-cdk/python/airbyte_cdk/sources/declarative/interpolation/filters.py

```diff
@@ -51,5 +51,43 @@ def hash(value, hash_type="md5", salt=None):
     return computed_hash
 
 
-_filters_list = [hash]
+def base64encode(value: str) -> str:
```

Comment:
Let's document this in [the docs and add an example](https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml#L2141). It would be worth adding a new section for filters similar to the macros section and documenting `hash` too while we're at it.

Reply:
I've added new `filters`section, please, check